package.json

```diff
@@ -24,48 +24,45 @@
     "testTypes": "tsc --noEmit --jsx react typeTest.tsx"
   },
   "peerDependencies": {
-    "react": ">=15 || ^0.14.7"
+    "react": "^16.8.0"
   },
   "dependencies": {
-    "@babel/runtime": "^7.2.0",
+    "@babel/runtime": "^7.9.6",
```

Comment:
Will this cause problems? Or will it just ensure that the runtime required by an end-user ends up being at least this recent?